lls/main.c

```diff
@@ -79,8 +145,14 @@ process_pkts(__attribute__((unused)) struct lls_config *lls_conf,
 		    !is_same_ether_addr(&eth_hdr->d_addr, &iface->eth_addr))
 			goto free_buf;
 
-		/* TODO Add support for ARP and ND packets. */
+		/* TODO Add support for ND packets. */
 		switch (rte_be_to_cpu_16(eth_hdr->ether_type)) {
+		case ETHER_TYPE_ARP:
+			if (process_arp(lls_conf, iface, tx_queue,
+					bufs[i], eth_hdr) == -1)
+				goto free_buf;
+			/* ARP reply was sent. */
```

Comment:
It's meant to explain why the `continue` is there and the buffer does not need to be freed, so I expanded the comment to explicitly say this.